### DISCUSSION TITLE
iPhone 16 Pro Max

### DISCUSSION THREAD

**Discussion by gcastle2007** [WLAN-Pi/wlanpi-profiler#222] [Q&A]
I profiled iPhone 16 Pro Max. There are many points around us that broadcast the regulatory domain RU.
Question: does the profiler parse information about 6GHz?
I have attached the profiler report and the pcap file.
[Uploading iPhone 16 Pro Max.zip…]()

**Comment by joshschmelzle** [comment]
The only way to profile in 6 GHz is to manually stage the interface on the target 6 GHz channel and then run profiler with the `--noprep` arg to prevent profiler from doing interface staging. It _might_ work, but is not heavily tested yet. Also, we still need to enhance the parser for Wi-Fi 7 features.

**Reply by gcastle2007** [reply]
Well, the task is not simple for me, so far I have not managed to launch an access point somehow with MT7925.
